apps/condo/domains/organization/integrations/sbbol/sync/syncBankAccounts.js

```diff
@@ -0,0 +1,83 @@
+const { BankAccount } = require('@condo/domains/banking/utils/serverSchema')
+const { RUSSIA_COUNTRY } = require('@condo/domains/common/constants/countries')
+const { ISO_CODES_FOR_SBBOL } = require('@condo/domains/common/constants/currencies')
+const { getSchemaCtx } = require('@open-condo/keystone/schema')
+const { Organization } = require('@condo/domains/organization/utils/serverSchema')
+
+const logger = getLogger('sbbol/syncBankAccounts')
+
+const sync = async (client) => {
+    const { keystone: userContext } = await getSchemaCtx('User')
+    const { keystone: bankAccountContext } = await getSchemaCtx('BankAccount')
+
+    const [organization] = await Organization.getAll(userContext,
+        { tin: client.inn },
+        { sortBy: 'createdAt_DESC' },
+    )
+
+    for (let accountIndex = 0; accountIndex <= client.accounts.length - 1; accountIndex++) {
+        const currentAccount = client.accounts[accountIndex]
+        const bankAccountDetails = {
+            tin: organization.tin,
+            country: RUSSIA_COUNTRY,
+            number: currentAccount.number,
+            currencyCode: (ISO_CODES_FOR_SBBOL[currentAccount.currencyCode]),
+            routingNumber: currentAccount.bic,
+        }
+        const foundAccount = await BankAccount.getOne(bankAccountContext, {
+            ...bankAccountDetails,
+            organization: { id_in: organization.id },
+        })
+        if (!foundAccount) {
+            await BankAccount.create(
+                bankAccountContext,
+                {
+                    meta: {
+                        sbbol: {
+                            type: currentAccount.type,
+                            state: currentAccount.state,
+                            openDate: currentAccount.openDate,
+                            name: currentAccount.name,
+                            closeDate: currentAccount.closeDate,
+                        },
+                    },
+                    ...bankAccountDetails,
+                    ...dvSenderFields,
+                    organization: { connect: { id: organization.id } },
+                }
+            )
+        }
+    }
+}
+/**
+ * Fetches ClientAccounts from SBBOL for specified date and creates BankAccount if this bank account has not yet been created
+ *
+ * @return {Promise<void>}
+ */
+const syncBankAccounts = async () => {
+
+    const fintechApi = await initSbbolFintechApi()
+    if (!fintechApi) return
+
+    logger.info('Checking, whether the user have ClientAccount items')
+
+    const response = await fintechApi.getClientAccounts()
+    const error = get(response, 'error')
+    if (error) return logger.error({ msg: 'getClientAccounts from SBBOL error', error })
+    const client = get(response, 'data')
+    const accounts = get(client, 'accounts', [])
+    if (accounts.length === 0) {
+        logger.info('SBBOL did not return any ClientAccount, do nothing')
+    } else {
+        sync(client)
```

Comment:
This is async function! You should use await 
```suggestion
        await sync(client)
```